lib/helper/resultHelper.py

```diff
@@ -70,14 +70,17 @@ def output_result(test_method_name,current_run_result, output_fp, time_list_coun
     with open(output_fp, "wb") as fh:
         json.dump(result, fh, indent=2)
 
-    with open(time_list_counter_fp, "w") as wfh:
-        wfh.write(str(len(result[test_method_name]['time_list'])))
-
+    # output sikuli status to static file
+    with open (time_list_counter_fp) as fh:
+        stat_data = json.load(fh)
+    with open(time_list_counter_fp, "w") as fh:
+        stat_data['time_list_counter'] = str(len(result[test_method_name]['time_list']))
+        json.dump(stat_data, fh)
```

Comment:
If you want to read and then write (overwrite existing contents), you can use `with open(time_list_counter_fp, 'r+') as fh:`.